imports/plugins/core/orders/client/components/OrdersTable.js

```diff
@@ -22,79 +20,163 @@ const useStyles = makeStyles({
+        // title: "Order Status",
+        options: [
+          // { label: "All", value: "" },
+          { label: i18next.t("admin.table.orderStatus.coreOrderWorkflow/canceled"), value: "canceled" },
+          { label: i18next.t("admin.table.orderStatus.coreOrderWorkflow/completed"), value: "completed" },
+          { label: i18next.t("admin.table.orderStatus.new"), value: "new" },
+          { label: i18next.t("admin.table.orderStatus.coreOrderWorkflow/processing"), value: "processing" }
+        ]
+      }),
+      show: false
+    },
+    {
+      Header: i18next.t("admin.table.headers.payment"),
+      accessor: (row) => row.payments[0].status,
+      id: "paymentStatus",
+      // eslint-disable-next-line react/no-multi-comp,react/display-name,react/prop-types
+      Cell: ({ row }) => <Fragment>{i18next.t(`admin.table.paymentStatus.${row.values.paymentStatus}`)}</Fragment>,
+      Filter: makeDataTableColumnFilter({
+        isMulti: true,
+        options: [
+          { label: i18next.t("admin.table.paymentStatus.completed"), value: "completed" },
+          { label: i18next.t("admin.table.paymentStatus.created"), value: "created" }
+        ]
+      })
     },
     {
-      Header: "Fulfillment",
-      accessor: "fulfillmentGroups[0].status",
-      Cell: ({ row }) => <Fragment>{i18next.t(`admin.table.fulfillmentStatus.${row.values["fulfillmentGroups[0].status"]}`)}</Fragment>
+      Header: i18next.t("admin.table.headers.fulfillment"),
+      accessor: (row) => row.fulfillmentGroups[0].status,
+      id: "fulfillmentStatus",
+      // eslint-disable-next-line react/no-multi-comp,react/display-name,react/prop-types
+      Cell: ({ row }) => <Fragment>{i18next.t(`admin.table.fulfillmentStatus.${row.values.fulfillmentStatus}`)}</Fragment>,
+      Filter: makeDataTableColumnFilter({
+        isMulti: true,
+        options: [
+          { label: i18next.t("admin.table.fulfillmentStatus.coreOrderWorkflow/completed"), value: "completed" },
+          { label: i18next.t("admin.table.fulfillmentStatus.new"), value: "new" },
+          { label: i18next.t("admin.table.fulfillmentStatus.coreOrderWorkflow/processing"), value: "processing" }
+        ]
+      })
     },
     {
-      Header: "Customer",
+      Header: i18next.t("admin.table.headers.customer"),
       accessor: "payments[0].billingAddress.fullName"
     },
     {
+      accessor: "payments[0].amount.displayAmount",
+      // eslint-disable-next-line react/no-multi-comp,react/display-name,react/prop-types
       Header: () => <Box textAlign="right">{i18next.t("admin.table.headers.total")}</Box>,
-      Cell: ({ cell }) => <Box textAlign="right">{cell.value}</Box>,
-      accessor: "payments[0].amount.displayAmount"
+      // eslint-disable-next-line react/no-multi-comp,react/display-name,react/prop-types
+      Cell: ({ cell }) => <Box textAlign="right">{cell.value}</Box>
+    },
+    {
+      // Hide this column, it's only for filtering
```

Comment:
Add `Header` so the name shows up in the sidebar; otherwise, it says "Filter" as the default.

![image](https://user-images.githubusercontent.com/42217/73405126-03911480-42a8-11ea-832a-0830eb0a53cf.png)

Reply:
added.